### ISSUE TITLE
Running falanx on a proto file with a map in it fails with error

### ISSUE THREAD

**Issue by 7sharp9** [jet/falanx#149]
```
An error occurred while generating type for message SomeMessageWithMap: System.ArgumentException: Incompatible record length
Parameter name: args
   at Microsoft.FSharp.Quotations.PatternsModule.mkNewRecord(Type ty, FSharpList`1 args)
   at Falanx.Proto.Core.JsonCodec.createLambdaRecord(TypeDescriptor typeDescriptor) in falanx/src/Falanx.Proto.Core/JsonCodec/Codec.fs:line 53
   at Falanx.Proto.Core.JsonCodec.createJsonObjCodec(TypeDescriptor typeDescriptor) in falanx/src/Falanx.Proto.Core/JsonCodec/Codec.fs:line 433
   at Falanx.Proto.Generator.TypeGeneration.createType@255-6.Invoke(Codec codec) in falanx/src/Falanx.Proto.Generator/TypeGeneration.fs:line 281
   at Microsoft.FSharp.Collections.SetTreeModule.iter[a](FSharpFunc`2 f, SetTree`1 t)
   at Falanx.Proto.Generator.TypeGeneration.createType(ProvidedTypeDefinition container, String scope, FSharpMap`2 lookup, FSharpSet`1 codecs, ProtoMessage message) in falanx/src/Falanx.Proto.Generator/TypeGeneration.fs:line 253

```

**Comment by 7sharp9** [comment]
proto file is maptest.proto:
```
syntax = "proto3";

message SomeMessage {
    int64 id = 1;
}

message SomeMessageWithMap {
    map<string, SomeMessage> some_map = 1;
}
```

**Comment by 7sharp9** [comment]
Fixed by #150
